src/tools/rustdoc-themes/main.rs

```diff
@@ -38,9 +38,12 @@ fn main() {
         eprintln!("No theme found in \"{}\"...", themes_folder);
         exit(1);
     }
+    let arg_name = "--check-theme".to_owned();
     let status = Command::new(rustdoc_bin)
-                        .args(&["-Z", "unstable-options", "--theme-checker"])
-                        .args(&themes)
+                        .args(&["-Z", "unstable-options"])
```

Comment:
Isn't this `unstable-options` unnecessary now that this option's being stabilized?

Reply:
It shouldn't anymore. Good catch!